src/generator/generator.ts

```diff
@@ -41,7 +41,8 @@ export async function protocolGen(host: Host) {
       host.WriteFile(`${filePrefix}${op.name.toLowerCase()}_client.go`, op.content, undefined, 'source-file-go');
     }
 
-    const constants = await generateConstants(session);
+    const version = await session.getValue("go-version", "<version>");
```

Comment:
Why not make the version parameter mandatory?  I don't a reason not to and it avoids us sending meaningless data of "<version>"

Reply:
The current workflow for management folks is to generate code and then modify it by hand. They requested this flag to improve that.

I wasn't sure if we wanted a "breaking" change like this for regeneration? Suggestions on who to ask if this is better?

Reply:
Are you referring to breaking in process or code?  I'll speak to both,
The telemetry string we send should reflect the version of the library that is running.  Thus it should rev which each revision of the library.  It's not considered breaking to change that string.

For process we should evolve that to where it should be (shift-left).  If that means we need to change (i.e. break process) to enforce constraints and increase the quality of generated code then we should do so.

Reply:
I was only concerned on breaking process. 

Sounds like we should make this mandatory.